asv.conf.json

```
@@ -1,32 +1,65 @@
 {
+    "version": 1,
+
     // The versions of Python to test against
-    "pythons": ["3.6"],
+    "pythons": ["3.8"], // Python 3.7 included below with extra cached_property dependency
 
-    "environment_type": "conda",
-    "conda_channels": ["conda-forge", "defaults"],
+    // "environment_type": "virtualenv",
+    "environment_type": "virtualenv",
```

Remove duplicated line:

```suggestion
    "environment_type": "virtualenv",
```